cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -599,30 +614,30 @@ func (p *BicepProvider) convertToDeployment(bicepTemplate BicepTemplate) (*Deplo
 }
 
 // Deploys the specified Bicep module and parameters with the selected provisioning scope (subscription vs resource group)
-func (p *BicepProvider) deployModule(ctx context.Context, scope infra.Scope, bicepPath string, parametersPath string) (*azcli.AzCliDeployment, error) {
+func (p *BicepProvider) deployModule(ctx context.Context, scope infra.Scope, bicepPath string, parametersPath string) (
+	result armresources.DeploymentExtended, err error) {
```

Comment:
I would prefer the return type to continue to be a pointer so we can return a `nil` value in the error case.

Reply:
sure, updated.